docs/en_US/api/restapi/configs.md

```diff
@@ -0,0 +1,21 @@
+# Dynamic Reload Configs
+
+By dynamically reloading configuration, parameters such as debug and timezone
```

Comment:
1. Add a link to basic configuration page: https://ekuiper.org/docs/en/latest/configuration/global_configurations.html
2. List the current supported configures.

Reply:
Added.